src/main.rs

```diff
@@ -1,8 +1,8 @@
+#![recursion_limit = "256"]
```

Comment:
This is needed for the test. In order to execute the json! reading with a lot of sub_objects, one needs to increase the recursion_limit.